tests/foreman/api/test_repositories.py

```diff
@@ -138,3 +142,28 @@ def test_positive_epel_repositories_with_mirroring_policy(
     )[0]
     assert repodata.content_counts['rpm'] != 0
     assert module_repo_options['mirroring_policy'] == repodata.mirroring_policy
+
+
+def test_negative_uplaod_expired_manifest(module_org, target_sat):
+    """Upload an expired manifest and attempt to refresh it
+
+    :id: d6e652d8-5f46-4d15-9191-d842466d45d0
+
+    :Steps:
+        1. Upload a manifest
+        2. Delete the Subscription Allocation on RHSM
+        3. Attempt to refresh the manifest
+
+    :expectedresults: Manifest refresh should fail and return error message
+    """
+    manifester = Manifester(manifest_category=settings.manifest.entitlement)
+    manifest = manifester.get_manifest()
+    manifester.delete_subscription_allocation()
+    upload_manifest(module_org.id, manifest.content)
```

Comment:
```suggestion
    target_sat.upload_manifest(module_org.id, manifest.content)
```
Lets use the target_sat if its present : )